catalogs/flink-iceberg-hive/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.5"
+
+services:
+
+  flink:
+    build: ./flink
+    depends_on:
+      - minio
+      - hive-metastore
+    ports:
+      - "8081:8081"
+
+  minio:
+    image: minio/minio
+    container_name: minio
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+    ports:
+      - 9001:9001
+      - 9000:9000
+    command: ["server", "/data", "--console-address", ":9001"]
+
+  mc:
+    depends_on:
+      - minio
+    image: minio/mc
+    container_name: mc
+    entrypoint: >
+      /bin/sh -c "
+      until (/usr/bin/mc config host add minio http://minio:9000 admin password) do echo '...waiting...' && sleep 1; done;
+      /usr/bin/mc rm -r --force minio/warehouse;
+      /usr/bin/mc mb minio/warehouse;
+      tail -f /dev/null
+      " 
+
+  hive-metastore:
+    container_name: hms
+    build: ./hms-standalone-s3
+    ports:
+      - "9083:9083"
+    environment:
+      - HMS_LOGLEVEL=INFO
+
+# Without a network explicitly defined, you hit this Hive/Thrift error
+# java.net.URISyntaxException Illegal character in hostname
+# https://github.com/TrivadisPF/platys-modern-data-platform/issues/231
+networks:
```

Comment:
Shouldn't the services above in this network then?

Reply:
I think it happens implicitly.